saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -225,3 +206,118 @@ def mutate(cls, root, info, id):
             type=OrderEvents.PLACED_FROM_DRAFT.value,
             user=info.context.user)
         return DraftOrderComplete(order=order)
+
+
+class DraftOrderLineAdd(BaseMutation):
```

Comment:
I wonder if this kind of mutations cannot be used for both draft and non-draft orders?

Reply:
Also, I think we are using `Create` instead of `Add` usually

Reply:
Adding order lines is valid only for draft orders. On branch `checkout-2.0` there is a similar mutation for adding lines to checkout, but I'd keep them separated.